tools/android/MASTG-TOOL-0121.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Apkleaks
+platform: android
+source: https://github.com/dwisiswant0/apkleaks
+---
+
+Apkleaks [https://github.com/dwisiswant0/apkleaks] is an open-source utility designed for static analysis of Android APK files, with a primary focus on identifying sensitive data such as API keys, URLs, AWS S3 buckets, and Firebase URLs. This tool automates the process of string analysis, facilitating the detection of hardcoded secrets and potential security vulnerabilities within Android applications.
+
+It offers support for custom regular expression rules, enabling users to specify additional search criteria through a JSON configuration file [regexes.json](https://github.com/dwisiswant0/apkleaks/blob/master/config/regexes.json). This adaptability allows for a customized analysis, thereby increasing its effectiveness in various security testing contexts. The tool integrates smoothly into testing workflows and provides clear, actionable insights to support efficient vulnerability management.
```

Comment:
Can you make this more objective / fact-based? Adjectives like "smoothly/clear/actionable/efficient" are very subjective/sales-y.

I would honestly just delete the second sentence, as it doesn't add any value.